### PR TITLE
Angular: enhanced workspace to be SEO ready

### PR DESCRIPTION
**Pull request by hollowaykeanho**
Since this new Angular/ workspace is ready for static site deployment, we can proceed to enhance it for SEO ready. Hence, let's do  this.

This patch enhances Angular/ workspace to be SEO ready.

### CODE REVIEW

File: Angular/.ci/test_unix-any.sh [hunk 1]
```diff
@@ -45,25 +45,13 @@ if [ $(OS_Is_Run_Simulated) -eq 0 ]; then
...
-                fi
-        done << EOF
-chromium
-google-chrome
-brave-browser
-EOF
-        IFS="$___old_IFS" && unset ___old_IFS ___line
-
-        if [ "$___browser" = "" ]; then
+        ___browser="$(type -p chromium)"
```
Comment (corygalyna) [inline Angular/.ci/test_unix-any.sh]: Remove this since the new `services/app/shell/init.*` are doing the job.

Comment (hollowaykeanho) [reply Angular/.ci/test_unix-any.sh]: issue fixed.

**Review by corygalyna — APPROVED** [review]

**Comment by corygalyna** [comment]
Looks good for me. You have to con!

**Comment by hollowaykeanho** [comment]
Thanks.

**Pull request merged**